packages/account/src/Sections/Security/DeactivateAccount/account-has-balance.jsx

```diff
@@ -145,9 +174,70 @@ const AccountHasBalanceOrOpenPositions = ({ details, mt5_login_list, client_acco
                         {mt5_balance.map(account => (
                             <Content
                                 key={account.login}
-                                currency_icon={`IcMt5-${getMT5Account(account.market_type, account.sub_account_type)}`}
+                                currency_icon={`IcMt5-${getCFDAccount({
+                                    market_type: account.market_type,
+                                    sub_account_type: account.sub_account_type,
+                                    platform: 'mt5',
+                                })}`}
+                                loginid={account.display_login}
+                                title={getCFDAccountDisplay({
+                                    market_type: account.market_type,
+                                    sub_account_type: account.sub_account_type,
+                                    platform: 'mt5',
```

Comment:
Can we define an object for platforms and reference `platform` prop from there for both `mt5` and `dxtrade`